website/docs/fundamentals/exposing/ingress/external-dns.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "External DNS"
+sidebar_position: 30
+---
+
+External-DNS is a Kubernetes controller that automatically manages DNS records for your cluster's services and ingresses. It acts as a bridge between Kubernetes resources and DNS providers like AWS Route 53, 
```

Comment:
Lets give a tiny bit of context about why you want a DNS entry for your load balancer.